core/src/banking_stage.rs

```diff
@@ -427,20 +413,11 @@ impl BankingStage {
                 batch_limit,
                 transaction_status_sender.clone(),
                 &gossip_vote_sender,
+                &mut buffered_packets,
             ) {
                 Err(RecvTimeoutError::Timeout) => (),
                 Err(RecvTimeoutError::Disconnected) => break,
-                Ok(mut unprocessed_packets) => {
-                    if unprocessed_packets.is_empty() {
-                        continue;
-                    }
-                    let num: usize = unprocessed_packets
-                        .iter()
-                        .map(|(_, unprocessed)| unprocessed.len())
-                        .sum();
-                    inc_new_counter_info!("banking_stage-buffered_packets", num);
```

Comment:
maybe this metrics isn't important anymore?

Reply:
yeah it's covered now by the `banking_stage-newly_buffered_packets_count` metric, but I think it's also not a bad idea to record the total buffered size in `banking_stage-total_buffered_packets_count`

Reply:
Aggregated all the metrics under BankingStageStats